src/main/java/io/kestra/plugin/cloudquery/Sync.java

```diff
@@ -0,0 +1,214 @@
+            title = "sync CloudQuery with yaml input",
+            code = {
+                "        env:",
+                "          AWS_ACCESS_KEY_ID: \"{{ secret('AWS_ACCESS_KEY_ID') }}\"",
+                "          AWS_SECRET_ACCESS_KEY: \"{{ secret('AWS_SECRET_ACCESS_KEY') }}\"",
+                "          AWS_DEFAULT_REGION: \"{{ secret('AWS_DEFAULT_REGION') }}\"    ",
+                "          PG_CONNECTION_STRING: \"postgresql://postgres:{{secret('DB_PASSWORD')}}@host.docker.internal:5432/demo?sslmode=disable\"",
+                "        configs:",
+                "           -",
+                "             kind: source",
+                "               spec:",
+                "                 name: aws",
+                "                 path: cloudquery/aws",
+                "                 version: \"v22.4.0\"",
+                "                 tables: [\"aws_s3*\", \"aws_ec2*\", \"aws_ecs*\", \"aws_iam*\", \"aws_glue*\", \"aws_dynamodb*\"]",
+                "                 destinations: [\"postgresql\"]",
+                "                 spec:",
+                "            -",
+                "              kind: destination",
+                "              spec:",
+                "                name: \"postgresql\"",
+                "                version: \"v5.0.3\"",
+                "                path: \"cloudquery/postgresql\"",
+                "                write_mode: \"overwrite-delete-stale\"",
+                "                spec:",
+                "                  connection_string: ${PG_CONNECTION_STRING}",
+            }
+        ),
+        @Example(
+            title = "sync CloudQuery with file input",
+            code = {
+                "        env:",
+                "          AWS_ACCESS_KEY_ID: \"{{ secret('AWS_ACCESS_KEY_ID') }}\"",
+                "          AWS_SECRET_ACCESS_KEY: \"{{ secret('AWS_SECRET_ACCESS_KEY') }}\"",
+                "          AWS_DEFAULT_REGION: \"{{ secret('AWS_DEFAULT_REGION') }}\"    ",
+                "          PG_CONNECTION_STRING: \"postgresql://postgres:{{secret('DB_PASSWORD')}}@host.docker.internal:5432/demo?sslmode=disable\"",
+                "        configs:",
+                "           - sources.yml",
+                "           - destination.yml",
+            }
+        ),
+    }
+)
+public class Sync extends AbstractCloudQueryCommand implements RunnableTask<ScriptOutput> {
+    private static final ObjectMapper OBJECT_MAPPER = JacksonMapper.ofYaml();
+    private static final String DB_FILENAME = "icrementaldb.sqlite";
+    private static final String CLOUD_QUERY_STATE = "CloudQueryState";
+
+    @Schema(
+        title = "CloudQuery configurations.",
+        description = "A list of CloudQuery configurations or files containing CloudQuery configurations"
+    )
+    @PluginProperty(
+        dynamic = false,
+        additionalProperties = String.class
+    )
+    @NotNull
+    private Object configs;
```

Comment:
```suggestion
    private List<Object> configs;
```
Then you can avoid checking if it's a list.